packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/react-native/installoktaauthsdk.md

```diff
@@ -0,0 +1,24 @@
+## Install dependency
+
+```bash
+npm install @okta/okta-react-native
+```
+
+## Configure the SDK
+
+Call the `createConfig` function to create a configured client when app initialized. You can put it in `index.js` or `App.js`. Resolves true if it successfully configures a client. 
+
+**Note:** requireHardwareBackedKeyStore is a configurable setting only on Android devices. If you're testing on Android emulators, set this field to false.
+
+```javascript
+import { createConfig } from '@okta/okta-react-native';
+
+createConfig({
+    clientId: "{clientId}",
```

Comment:
I see a mixed use of single quotes and double quotes in this code snippet. Should we use one or the other?